spec/requests/vendor_api/post_test_data_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+RSpec.describe 'Vendor API - GET /api/v1/test-data/regenerate', type: :request do
```

Comment:
```suggestion
describe 'Vendor API - GET /api/v1/test-data/regenerate', type: :request do
```

It's not needed, is it?

Reply:
FWIW I prefer the explicit `RSpec.` Not entirely sure why — probably paranoia

Reply:
I too have a desire to keep our monkeys unpatched! I will [update the corresponding config option](https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/master/spec/spec_helper.rb#L74-L79) in a follow up PR.